### PR TITLE
Drop unused test_files directive from gemspec

### PR DESCRIPTION
**Pull request by olleolleolle**
The test_files directive is not used for anything in RubyGems.org any longer.

### CODE REVIEW

**Comment by justincampbell** [comment]
@olleolleolle I added your public GitHub email as a gem owner. You can email me if you need it via a different email. Thanks!

**Comment by olleolleolle** [comment]
@justincampbell Oh, that's what happens to a fly that flies too close to the candle.

**Comment by olleolleolle** [comment]
@justincampbell The default branch is still protected, so all commits needs at least 1 approving review from another maintainer.

(I don't have permissions to change any of that.)

**Review by justincampbell — APPROVED** [review]

**Pull request merged**
